pkg/template/resource_test.go

```diff
@@ -469,6 +470,33 @@ func Test_ResolveTrigger(t *testing.T) {
 			TriggerTemplate: &tt,
 		},
 	},
+		{
+			name: "embedded trigger template",
+			trigger: triggersv1.EventListenerTrigger{
+				Template: &triggersv1.EventListenerTemplate{
+					Spec: &triggersv1.TriggerTemplateSpec{
+						ResourceTemplates: []triggersv1.TriggerResourceTemplate{{
+							RawExtension: runtime.RawExtension{
+								Raw: []byte(`{"kind":"PipelineRun","apiVersion":"tekton.dev/v1alpha1","metadata":{},"spec":{},"status":{}}`),
```

Comment:
Similar RawExtension func could be useful here as well.

Reply:
done